localize-dynamic-mixin: Don't request unsupported langpacks in dev (#1576)
BrightspaceUI/core: mixins/localize-mixin.md

}
```

> When using this method, depending on various user settings, it's possible that a language file that does not exist will be requested, resulting in a network error (404). In production, your build system should prevent this by transpiling the variable dynamic import. BSI does this for you.

Reviewer: I'd rather not reference BSI in core if we can avoid it, since core is really more uh... "core" than BSI and could be used by things other than Brightspace. BSI also may not always be around (fingers crossed).

```suggestion
When using this method, depending on various user settings, it's possible that a language file that does not exist will be requested, resulting in a network error (404). In production, your build system should prevent this by transpiling the variable dynamic import into a `switch` statement.
```
